test/features/events/response.test.ts

```diff
@@ -204,7 +203,7 @@ test('XMLHttpRequest: emits the "response" event upon the original response', as
       headers: headersContaining({
         'x-request-custom': 'yes',
       }),
-      credentials: 'same-origin',
+      credentials: 'omit',
```

Comment:
XHR is polyfilled by `http`. In `http`, `credentials` is set to `same-origin`. In XHR, however, credentials depends on the `this.withCredentials` property, and is either `include` or `omit`.